Fixing the corfu_add_node cmdlet

rogermichoud opened the pull request:
## Overview
Reflecting new api of addNode that adds timeout
and poll frequency of state of the workflow.


Why should this be merged: 
Blocks fault-injection testing

## Checklist (Definition of Done):

- [X] There are no TODOs left in the code
- [X] [Coding conventions](https://github.com/CorfuDB/CorfuDB/wiki/Corfu-Style-Guidelines) (e.g. for logging, unit tests) have been followed
- [X] Change is covered by automated tests
- [X] Public API has Javadoc

no2chem commented on corfu_scripts/corfu_add_node.clj:
```diff
@@ -23,7 +25,8 @@ Options:
 (defn add-node [endpoint]
-      (println (.. (get-management-view) (addNode endpoint))))
+      (println (.. (get-management-view)
+                   (addNode endpoint 3 (. Duration (ofMinutes 10)) (. Duration (ofSeconds 1))))))
```
Shouldn't these parameters come from the docopt string?

Maithem commented:
can you add remove node as well

rogermichoud commented:
@no2chem Addressed the comments (Damn it was painful) 

@Maithem Test cycle for cmdlets is too painful and I need more background on how to use Clojure to become somewhat efficient. In the meantime, I will defer your ask to a new pull request. We need this add_node fix for deblocking our testing. Remove node is a bit less urgent.

Maithem commented:
@rogermichoud ok that's fine, I thought you wanted both for the tests.
